src/components/LoginForm/Form.js

```diff
@@ -0,0 +1,113 @@
+import { Button, FormHelperText, Grid } from "@material-ui/core";
+import { ErrorMessage, Form as FForm } from "formik";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import Input from "@/promisetracker/components/Form/Input";
+
+function Form({
+  disabled,
+  errors,
+  fields,
+  name: nameProp,
+  onSubmit,
+  status,
+  ...props
+}) {
+  const classes = useStyles(props);
+  const name = nameProp || "individual-registration-form";
+
+  return (
+    <FForm className={classes.root}>
+      <Grid container>
+        <Grid item xs={12}>
+          <Input
+            inputProps={{ className: classes.inputProps }}
+            className={classes.input}
+            id={`${name}-email`}
+            name="email"
+            type="email"
+            placeholder={fields?.email?.placeholder}
+          />
+          <FormHelperText
+            id={`${name}-email-helper-text`}
+            error={errors?.email}
+          >
+            <ErrorMessage name="email" />
+          </FormHelperText>
+        </Grid>
+        <Grid item xs={12}>
+          <Input
+            inputProps={{ className: classes.inputProps }}
+            className={classes.input}
+            id={`${name}-password`}
+            name="password"
+            type="password"
+            placeholder={fields?.password?.placeholder}
+          />
+          <FormHelperText
+            id={`${name}-password-helper-text`}
+            error={errors?.password}
+          >
+            <ErrorMessage name="password" />
+          </FormHelperText>
+        </Grid>
+        <Grid item xs={12}>
+          <FormHelperText
+            className={!status.error ? classes.success : ""}
+            id={`${name}-status-helper-text`}
+            {...status}
+          />
+          <Button
+            variant="contained"
+            color="secondary"
+            disabled={disabled}
+            onClick={onSubmit}
+            className={classes.authButton}
+          >
+            {fields?.submit?.label}
+          </Button>
```

Comment:
Why are we disabling the button? I don't even get the pointer mouse when I hover on the button.

Reply:
if the field isn't valid, I disable the button.
once i get the right data it get enabled
![image](https://user-images.githubusercontent.com/23483005/151127575-cc04eef7-bba8-4a12-b9eb-9fcf5362bcee.png)
Unless you think the ash background should be the state of the button all the time.

Reply:
Nah, of Firefox, the page loads with button already disabled @KobbyMmo
![Peek 2022-01-26 11-48](https://user-images.githubusercontent.com/1779590/151131598-e64f0988-0bb3-49e4-8805-f755ff94485f.gif)